packages/react-google-analytics/src/GaJS.tsx

```diff
@@ -12,7 +12,7 @@ export interface Props {
   allowHash?: boolean;
   set?: any[][];
   onLoad?(analytics: GaJSAnalytics): void;
-  debug?: boolean;
```

Comment:
So this would be a breaking change (major bump) I guess?

Reply:
I think that makes sense. 👍